### DISCUSSION TITLE
Will the web editor be open sourced?

### DISCUSSION THREAD

**Discussion by pbsds** [typst/typst#152] [Q&A] [answered]
Overleaf is self-hostable, it would be great if typst could also be self-hosted.

**Comment by jason-s** [comment]
or at least available for free for non-commercial use & at a reasonable price for commercial use

**Comment by sascha-wi** [comment]
2 Years no answer.

**Reply by pbsds** [reply]
Makes sense if you think of typst as an open core product. The web interface is the money maker, at best they might sell on-prem hosting in the future

**Comment by laurmaedje** [comment, marked as answer]
It will not, see https://github.com/typst/webapp-issues?tab=readme-ov-file#faq
